examples/apps/lex-make-appointment/lambda_function.py

```diff
@@ -1,214 +0,0 @@
-'''
```

Comment:
Why is this file getting removed? The template still seems to reference it. Same question for some other apps here.

Reply:
The template in this folder seems to only references a different node.js file and never uses this python file. Could you specify which template you're talking about?

Reply:
Look at the directory structure. This app is found at `examples/apps/lex-make-appointment/` and is the js version.

Reply:
Oh haha I see now. I was looking at the lex-make-appointment-python template, not lex-make-appointment. Thanks for clarifying!